### PR TITLE
minimal Pg 14beta1 - update

### PR DESCRIPTION
**Pull request by ImreSamu**
as I see:

"PostgreSQL 14 Beta 1 Released!"  ( 2021-05-20  )
https://www.postgresql.org/about/news/postgresql-14-beta-1-released-2213/

this is a minimal test - for adding
IMHO: the next debian version ( "bullseye" )  selected.  ( for a [postgis dockerimage](https://github.com/postgis/docker-postgis)  - it has a newer geos version  )

### CODE REVIEW

File: 14/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+RUN mkdir -p /var/run/postgresql && chown -R postgres:postgres /var/run/postgresql && chmod 2777 /var/run/postgresql
+
+ENV PATH $PATH:/usr/lib/postgresql/$PG_MAJOR/bin
+ENV PGDATA /var/lib/postgresql/data
+# this 777 will be replaced by 700 at runtime (allows semi-arbitrary "--user" values)
+RUN mkdir -p "$PGDATA" && chown -R postgres:postgres "$PGDATA" && chmod 777 "$PGDATA"
+VOLUME /var/lib/postgresql/data
+
+COPY docker-entrypoint.sh /usr/local/bin/
+RUN ln -s usr/local/bin/docker-entrypoint.sh / # backwards compat
```
Comment (casio) [inline 14/Dockerfile:184]: Lil typo?
```suggestion
RUN ln -s /usr/local/bin/docker-entrypoint.sh / # backwards compat
```

Comment (tianon) [reply 14/Dockerfile:184]: Looks like `update.sh` wasn't run: https://github.com/docker-library/postgres/blob/376f87ce3b00273c5ea1f5446d6876227d5ddf07/update.sh#L159-L161

**Comment by tianon** [comment]
Do we have any idea what the GA timeline for 14 is going to be?  I'm not sure when Bullseye will GA, but we don't want to jump the gun too much there.

**Comment by ImreSamu** [comment]
> Do we have any idea what the GA timeline for 14 is going to be?

2021Q3?
_"The next major release of PostgreSQL is planned to be the 14 release. A tentative schedule for this version has a release in the third quarter of 2021."_    https://www.postgresql.org/developer/roadmap/

> I'm not sure when Bullseye will GA,

OK;    Let's go now with  "buster"  ..  and change later to "bullseye" 

( You can adapt|remove|drop my PR .. as you like :smile:   )

**Comment by ImreSamu** [comment]
Thanks for the fix! 

------------------

ouch .. as I see the PG14 (alpine) has new feature options 

`--with-lz4`
   _"Build with LZ4 compression support. This allows the use of LZ4 for compression of table data."_
( https://www.postgresql.org/docs/14/install-procedure.html )

probably important for the new **"LZ4 compression for TOAST"**
* https://blog.dbi-services.com/postgresql-14-lz4-compression-for-toast/

**Pull request closed without merging**